gensim/models/ldamodel.py

```diff
@@ -579,6 +579,15 @@ def update(self, corpus, chunksize=None, decay=None, offset=None,
         if gamma_threshold is None:
             gamma_threshold = self.gamma_threshold
 
+        if isinstance(corpus, list):
+            logger.warning(
```

Comment:
It looks like this
```
taking plain list as a corpus for LDA;we strongly recommend you to use corpus streaming instead of lists;LDA is processing the corpus in batches andit only loads one batch of documents into memory at once;please take a look at the tutorial on corpus streaming: https://radimrehurek.com/gensim/tut1.html#corpus-streaming-one-document-at-a-time
```